spectator/stateless_meters.h

```diff
@@ -6,14 +6,43 @@
 namespace spectator {
 
 namespace detail {
+
 inline std::string as_string(std::string_view v) { 
     return {v.data(), v.size()}; 
 }
+
+inline bool is_valid_character(char c) {
```

Comment:
I suggest using a table lookup to avoid the comparisons. For example, we initialize a valid_chars table (using constexpr to do it at compile time) and then do a simple check like what we do in spectator's AsciiSet: https://github.com/Netflix/spectator/blob/main/spectator-api/src/main/java/com/netflix/spectator/impl/AsciiSet.java#L184-L186

Reply:
Added a lookup table, borrowed from spectatord.